ui/raidboss/data/07-dt/raid/r11n.ts

```diff
@@ -0,0 +1,322 @@
+      alertText: (_data, matches, output) => {
+        const nextWeapon = weaponModelIDMap[matches.param1];
+        if (nextWeapon === 'axe')
+          return output.axe!();
+        if (nextWeapon === 'scythe')
+          return output.scythe!();
+        if (nextWeapon === 'sword')
+          return output.sword!();
+        return output.unknown!();
+      },
+      outputStrings: {
+        axe: {
+          en: 'Out next',
+        },
+        scythe: {
+          en: 'In next',
+        },
+        sword: {
+          en: 'Intercards next',
+        },
+        unknown: Outputs.unknown,
+      },
+    },
+    {
+      id: 'R11N One And Only',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3DC', source: 'The Tyrant', capture: true },
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R11N Cosmic Kiss', // Meteor towers
+      type: 'StartsUsing',
+      netRegex: { id: 'B3DE', source: 'Comet', capture: false },
+      suppressSeconds: 1,
+      response: Responses.getTowers(),
+    },
+    {
+      id: 'R11N Foregone Fatality',
+      type: 'Tether',
+      netRegex: { id: tetherData['foregoneTether'], capture: false },
+      suppressSeconds: 1,
+      alertText: (_data, _matches, output) => output.tetherBusters!(),
+      outputStrings: {
+        tetherBusters: Outputs.tetherBusters,
+      },
+    },
+    {
+      id: 'R11N Massive Meteor',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData['massiveMeteor'], capture: true },
+      condition: (data) => data.role !== 'tank',
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'R11N Double Tyrannhilation',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3E5', source: 'The Tyrant', capture: false },
+      alertText: (_data, _matches, output) => output.losMeteor!(),
+      outputStrings: {
+        losMeteor: {
+          en: 'LoS behind 2x meteor',
+        },
+      },
+    },
+    {
+      id: 'R11N Flatliner',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3E8', source: 'The Tyrant', capture: false },
+      infoText: (_data, _matches, output) => output.flatliner!(),
+      outputStrings: {
+        flatliner: {
+          en: 'Short knockback to sides',
+        },
+      },
+    },
+    {
+      id: 'R11N Majestic Meteor',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3E9', source: 'The Tyrant', capture: false },
+      infoText: (_data, _matches, output) => output.baitPuddles!(),
+      outputStrings: {
+        baitPuddles: {
+          en: 'Bait 3x puddles',
+        },
+      },
+    },
+    {
+      id: 'R11N Mammoth Meteor',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3EC', source: 'The Tyrant', capture: false },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.proxAOE!(),
+      outputStrings: {
+        proxAOE: {
+          en: 'Proximity AoE',
+        },
+      },
+    },
```

Comment:
Does it make sense to collect both of these casts and indicate safe intercards? (no strong opinion here)

Reply:
Also there's no `KB Towers` trigger for the 4 towers that spawn after the first set of these meteors